source/extensions/filters/http/common/aws/BUILD

```diff
@@ -0,0 +1,47 @@
+licenses(["notice"])  # Apache 2
+
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_library",
+    "envoy_package",
+)
+
+envoy_package()
+
+envoy_cc_library(
+    name = "signer_lib",
```

Comment:
Nit: if this is going to only ever be a pure interface, `signer_interface` would be more consistent with other conventions.

Reply:
Good to know. I thought that convention was only used the public headers. e.g. `include/envoy`.